style.css

```diff
@@ -115,26 +118,69 @@ html, body {
 .header__link {
     text-transform: uppercase;
     font-size: 12px;
+    letter-spacing: 1px;
     text-decoration: none;
-    color: #000000;
+    color: var(--main-text-color);
     font-family: 'Montserrat', sans-serif;
-    font-weight: bold;
+    cursor: pointer;
+}
+.header__link:hover, .header__cartlink:hover, .header__sublink:hover {
+    color: #a0522d;
 }
 .header__cartlink {
     font-size: 12px;
-    text-decoration: none;
-    color: #000000;
+    text-decoration: none;  
+    color: var(--main-text-color);
     font-family: 'Poppins', sans-serif;
     font-weight: bold;
     text-align: center;
- }
+}
+.form-search {
+    display: block;
+}
+.form-search input {
+    all: unset;
+    font-size: 10px;
+    letter-spacing: 2px;
+    width: 80%;
+}
+.form-search input[type="image"] {
+    width: 15px;
+    height: 15px;
+}
+.header__subnav {
+    display: none;
+}
+.header__subnav.active {
```

Comment:
why not to make a subnav styling initially and then `hide` it

Reply:
I don't know. That looked more logical to me at the time when i started.

Reply:
I suggest you to revert a class to be initially visible and add an a class to disable it, for example if I want to create a element and apply .header__subnav class and I dont know about it should have .active class as well I could need more time to research what is wrong with class

Reply:
I suggest you to revert a class to be initially visible and add an a class to disable it, for example if I want to create a element and apply .header__subnav class and I dont know about it should have .active class as well I could need more time to research what is wrong with class

Reply:
Ok. I switched classes and their visibility. And .header__cartlist too

Reply:
Initially I wanted that your elements would bve visible without second class 
So with class `header__subnav` I want to show header subnav and when i put next to class `header__subnav` class `header__subnav--disabled` or `header__subnav--hidden` it will set it to display none  
So the very popular case scenario is 
`class="header__subnav"` and when you want it not to be shown - `class="header__subnav header__subnav--hidden"`

Reply:
yes, maybe it would be easier that way )))